cql3/query_processor.cc

```diff
@@ -576,6 +576,8 @@ query_processor::execute_direct_without_checking_exception_message(const sstring
         if (!queryState.getClientState().isInternal)
             metrics.regularStatementsExecuted.inc();
 #endif
+    auto user = query_state.get_client_state().user();
+    tracing::trace(query_state.get_trace_state(), "Authenticated user: {}", user ? (user->name ? *user->name : "anonymous") : "no user authenticated");
     tracing::trace(query_state.get_trace_state(), "Processing a statement");
```

Comment:
No point in two messages here, we can merge them:
`Processing a statement for authenticated user: {}`

Reply:
merged